lib/utils.bash

```diff
@@ -190,7 +229,8 @@ find_versions() {
   version=$(get_version_from_env "$plugin_name")
   if [ -n "$version" ]; then
     local upcase_name
-    upcase_name=$(printf "%s\n" "$plugin_name" | tr '[:lower:]-' '[:upper:]_')
+    tr_args=("$plugin_name" "[:lower:]" "[:upper:]" "-" "_")
+    upcase_name=$(fast_tr ${tr_args[@]})
```

Comment:
Due to the subshell this is at best equally as fast as before. Probably a way to do this without a subshell.